rts/lra/service-base/src/main/java/io/narayana/lra/logging/LraI18nLogger.java

```diff
@@ -165,6 +165,9 @@ String info_failedToEnlistingLRANotFound(URL lraId, URI coordinatorUri, int coor
     @Message(id = 25039, value = "Invalid argument passed to method: %s")
     String error_invalidArgument(String reason);
 
+    @LogMessage(level = ERROR)
```

Comment:
The way the message is used implies that it should be logged at `level = WARN`.

Reply:
My mistake, sorry. I'll update it.